integration/client.go

```diff
@@ -22,3 +28,36 @@ func getK8sClient() (kubernetes.Interface, error) {
 
 	return cs, nil
 }
+
+type AWSClient struct {
+	EC2 *ec2.EC2
+}
+
+type AWSConfig struct {
+	AccessKeyID     string
+	AccessKeySecret string
+	SessionToken    string
+	Region          string
+}
+
+func getAWSClient(config *AWSConfig) AWSClient {
```

Comment:
Why this takes an argument when it's only called with `nil`? Is there a reason why this functions starts `get` instead of `new`? This is misleading because on every call new ec2 instance is created.

Reply:
good point, i've removed the config altogether, PTAL